src/style/themes/base/base-theme.config.js

```diff
@@ -35,7 +35,8 @@ export default (palette) => {
     disabled: {
       border: palette.slateTint(80),
       disabled: blackWithOpacity(0.55),
-      input: palette.slateTint(95)
+      input: palette.slateTint(95),
+      background: palette.slateTint(95)
```

Comment:
Is this the best name for this colour? Is it used for anything other than the dialogue?

Reply:
For now , it is used to the `Dialog`, what do you think about backgroundDialog?

Reply:
![Zrzut ekranu 2019-04-5 o 12 57 05](https://user-images.githubusercontent.com/12142840/55623117-568e5e00-57a2-11e9-894d-2dbdd0c36c7a.png)

Reply:
I propose add:
```
background: {
   primary: palette.slateTint(95)
}
```
and remove:
```
      input: palette.slateTint(95),
      background: palette.slateTint(95)
```

Reply:
I decided to move the color to the component when the color is in use